packages/sign_in_with_apple/README.md

```diff
@@ -30,9 +31,9 @@ SignInWithAppleButton(
 
 ## Flow
```

Comment:
We should potentially differentiate between iOS/macOS and Android here, but I think we can also revamp the README in another PR